woocommerce-connect-client.php

```diff
@@ -101,6 +120,14 @@ public function set_service_settings_store( WC_Connect_Service_Settings_Store $s
 			$this->service_settings_store = $settings_store;
 		}
 
+		public function get_tracks() {
```

Comment:
it wasn't strictly needed but all the other classes in the plugin have this and I wanted to remain consistent